app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -163,5 +163,24 @@ def institution_details(qualification)
     def personal_statement
       "Why do you want to become a teacher?: #{application_form.becoming_a_teacher} \n What is your subject knowledge?: #{application_form.subject_knowledge}"
     end
+
+    def offer
+      offered_option = application_choice.offered_course_option
+      offered_course = if offered_option.nil?
+                         {}
+                       else
+                         {
+                           course: {
+                              provider_code: offered_option.course.provider.code,
+                              course_code: offered_option.course.code,
+                              site_code: offered_option.site.code,
+                           },
+                         }
+                       end
+
+      {
+        conditions: application_choice.offer && application_choice.offer['conditions'],
+      }.merge(offered_course)
```

Comment:
- Maybe the `#course` method could be refactored to take a `course` param and could be reused here? (Hopefully this will keep us up to date with incoming changes to full/part time and cycle year too)
- given we're going on the basis that `offered_option` will always be populated if the offer is present (which seems right), when `offered_option` is `nil` here (ie no offer has been received) then I think it doesn't make sense to include a hash with only an empty `conditions` key because that implies an unconditional offer

Reply:
Great suggestion 👍

Reply:
Committed, tests updated!